Ejercicios parte 2

Chilpa19 opened the pull request:
Estos son la segunda parte de los ejercicios

Chilpa19 committed:
Entrega de ejercios parte 1

Se hizo la entrega de los 3 ejercicos de la parte 1

Chilpa19 committed:
Ejercicios 2

Se agregaron los otros 3 ejercicios de la tarea

Chilpa19 committed:
Update alan_chilpa_43_1.sh

JohnKun136NVCP commented on Shell-Script/Parte_1/Entregas/alan_chilpa_43_2/alan_chilpa_43_1.sh:
```diff
@@ -0,0 +1,27 @@
+	read opc
+	case $opc in
+		1)
+		USB=$(ls -A '/media/chilpalan/') 
```
El script tiene ser de manera general, no solamente en tu equipo.

JohnKun136NVCP reviewed (commented):
❗ Ya hice las observaciones.

Chilpa19 committed:
Codigo actualizado y modificado

Se modificaron los errores

Chilpa19 committed:
Codigo modificado

En la modificacion anterior estaba mal el nombre del archivo, este es el nueno xD

JohnKun136NVCP reviewed (approved):
🙆🏼 Ok.
